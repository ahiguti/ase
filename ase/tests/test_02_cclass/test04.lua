
local Test = ASE.CPP['ASETestCPP']
local o1 = Test.Create()
local o2 = Test.Create()
o1.ObjRef(o2);
o1.ObjCRef(o2);
o1.VariantRef("AA");
o1.VariantCRef("BBB");
