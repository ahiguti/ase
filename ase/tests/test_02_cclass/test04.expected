ObjRef: o=0 s=0
ObjCRef: o=0 s=0
VRef: s=AA
VCRef: s=BBB
